src/Components/Instagram-authentication.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import {
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import { auth } from "../firebase";
+
+export default class InstagramAuth extends React.Component {
```

Comment:
let's make the component filenames carry the exact same naming as the component itself, in capital letter as well